gammapy/data/tests/test_event_list.py

```diff
@@ -11,6 +11,20 @@
 from gammapy.utils.testing import mpl_plot_check, requires_data
 
 
+class TestEventListBasic:
+    def setup_class(self):
+        table = Table()
+        table["RA"] = [0.0, 0.0, 0.0, 10.0] * u.deg
+        table["DEC"] = [0.0, 0.9, 10.0, 10.0] * u.deg
+        table["ENERGY"] = [1.0, 1.5, 1.5, 10.0] * u.TeV
+        table["OFFSET"] = [0.1, 0.5, 1.0, 1.5] * u.deg
```

Comment:
I think the table misses a "TIME" column.

Reply:
Probably the constructor should test that the required columns are present, but I don't know what things are actually required; I took this construction from the setup function in the `TestEventSelection` and all tests pass.

Reply:
@Tobychev : You can find the mandatory columns here : https://gamma-astro-data-formats.readthedocs.io/en/v0.3/events/events.html#mandatory-columns.
I think just adding a TIME will suffice